src/interfaces/taskForToday.ts

```diff
@@ -0,0 +1,11 @@
+import { IStatus } from './status';
+
+export interface ITaskForToday {
+  id: number;
+  description: string;
+  status: IStatus;
+}
+
+export type getTaskForTodayType = {
```

Comment:
The type for functions will look like:
`type getTaskForTodayType = (attrs...) => returnedType;`